src/main/java/org/launchcode/lifeorganizer/models/dto/OptionFormDTO.java

```diff
@@ -0,0 +1,51 @@
+package org.launchcode.lifeorganizer.models.dto;
+
+import org.launchcode.lifeorganizer.models.User;
+
+import javax.validation.constraints.NotBlank;
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.Size;
+
+public class OptionFormDTO {
```

Comment:
I think this class should extend SignupFormDTO.java since we already have all the required fields.